bin/main

```diff
@@ -1,111 +1,130 @@
 #!/usr/bin/env ruby
-def disply_board(array)
-  position = 0
-  while position < array.length
-    if (position % 3).zero?
-      puts ''
-      print "---+---+---+\n"
+require_relative '../lib/logic'
+class Game
```

Comment:
- [ ] All game-logic related resources (classes and/or modules) are kept in the `lib` directory. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/lib_dir_exist.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))